ydb/tests/library/harness/kikimr_config.py

```diff
@@ -432,7 +432,32 @@ def __init__(
 
             self.yaml_config["feature_flags"]["enable_external_data_sources"] = True
             self.yaml_config["feature_flags"]["enable_script_execution_operations"] = True
-
+            
+        if os.getenv("YDB_YAML_CONFIG") is not None:      
```

Comment:
It will be better to move the merge code from the generator constructor to deploy function within `cmds` dir. Because merge is not part of config generator